snap/hooks/default-configure

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -e
+
+TAG="$SNAP_NAME.default-configure"
+
+logger --tag=$TAG "Start"
+
+autostart="$(snapctl get autostart)"
+
+if [ -z "$autostart" ]; then
+    echo "'autostart' is unset."
+elif [ "$autostart" = true ]; then
+    echo "'autostart' is enabled ($autostart). Starting and enabling openthread-border-router snap."
```

Comment:
```suggestion
    echo "'autostart' is enabled ($autostart): starting and enabling openthread-border-router services."
```
The snap itself isn't enabled/disable here. Enabling a snap (`snap enable <snap>`) serves a different purpose.